src/index.js

```diff
@@ -15,6 +15,25 @@ const validateFunctionOrUndefined = (param, paramName) => {
   if (!(typeof param === 'undefined' || typeof param === 'function')) throw new Error(`Please provide correct value for ${paramName}`)
 }
 
+const handleBulkResponseErrors = (bulkResponse, body) => {
+  if (bulkResponse.errors) {
+    const erroredDocuments = []
+    bulkResponse.items.forEach((action, i) => {
+      const operation = Object.keys(action)[0]
+      if (action[operation].error) {
+        erroredDocuments.push({
+          status: action[operation].status,
+          error: action[operation].error,
+          operation: body[i * 2],
```

Comment:
It would be good to add a test for these indexed parts of object, so if someone in the future will change something we will be noticed.

Reply:
Yep fair point; let me see if really really need this, I'll add tests if so otherwise remove. 
(it's taken directly form ES website docs btw)